admin/google_search_console/class-crawl-issue.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * @package WPSEO\Admin|Google_Search_Console
+ */
+
+/**
+ * Class WPSEO_Crawl_Issue
+ *
+ * @todo Might want to create a class that accepts the raw Google response as construct parameter.
```

Comment:
Why not create a static function on this function called `createFromGoogleResponse` that does just that?